index.yaml

```diff
@@ -1,9 +1,85 @@
 apiVersion: v1
 entries:
   vsecm:
+  - apiVersion: v2
+    appVersion: 0.22.5
+    created: "2024-02-26T19:20:33.786787-08:00"
+    dependencies:
+    - condition: global.deploySpire
+      name: spire
+      repository: file://charts/spire
+      version: 0.22.5
+    - condition: global.deploySafe
+      name: safe
+      repository: file://charts/safe
+      version: 0.22.5
+    - condition: global.deploySentinel
+      name: sentinel
+      repository: file://charts/sentinel
+      version: 0.22.5
+    description: Helm chart for VMware Secrets Manager
+    digest: f0c336973383b7adc0826389538dc4854eae7c6b9dce1f8782cbcad31281cbcc
+    home: https://vsecm.com/
+    icon: https://vsecm.com/assets/vsecm-256.png
+    keywords:
+    - secrets
+    - kubernetes
+    - secrets-manager
+    - spire
+    - spiffe
+    - zero-trust
+    - cloud-native
+    - edge
+    - secret-management
+    - security
+    name: vsecm
+    sources:
+    - https://github.com/vmware-tanzu/secrets-manager
+    type: application
+    urls:
+    - vsecm-0.22.5.tgz
+    version: 0.22.5
+  - apiVersion: v2
+    appVersion: 0.22.5-nocrd
```

Comment:
the `-nocrd` charts do not have `crds` folder.

that is useful when the end user manages all their CRDs in a separate place and duplicating crds cause issues.